packages/codesandbox-theme-docs/dist/index.js

```diff
@@ -495,7 +495,12 @@ var getGitEditUrl = (filePath) => {
   const repo = gitUrlParse(config.docsRepositoryBase || "");
   if (!repo)
     throw new Error("Invalid `docsRepositoryBase` URL!");
-  return `${repo.href}/${filePath}`;
+  console.log(repo.href);
```

Comment:
console leftover?

Reply:
Nice catch. Removed here: [ab01e68](https://github.com/codesandbox/docs/pull/101/commits/ab01e68ed3c76099f95cbbb2721b41bb82671e4d). I think we should take some time to add formatting and linting to this project.